### DISCUSSION TITLE
[Features] show path where soft [package] installed

### DISCUSSION THREAD

**Discussion by Zeroes1** [Devolutions/UniGetUI#2849] [General]
It's possible add features for show path where soft [package] installed?

i check latest 3.1.2-beta4 but not see this feature...
make issue for this?

**Comment by marticliment** [comment]
You can right-click a package -> open install location

**Comment by Zeroes1** [comment]
@marticliment ahhh yes i see now.

but this moment have troubles?
I can't see path sometime? 
menu item "Open install location" sometime disabled...

reason: 
1) install package on another user (no have admin right?)
2) bug logic
3) no have data for this
?

For example:

Windows 10 22h2, Uniget 3.1.2.0 Relis
Bitvise installed 18.10.2024 under admin, as same run uniget

![image](https://github.com/user-attachments/assets/4382562f-9fe8-4fa2-8d72-2fd317a25a9c)
Why i can't see path?

**Reply by marticliment** [reply]
Some package managers don't report the install location for a package, and it is determined purely by heuristics. When UniGetUI can't "guess" which the location is, the option shows greyed out
